core/main.py

```diff
@@ -0,0 +1,27 @@
+import logging
+
+# logging
+from asgi_correlation_id import CorrelationIdMiddleware
+from fastapi import FastAPI
+from fastapi.exception_handlers import http_exception_handler
+from fastapi.exceptions import HTTPException
+
+from core.configure_logging import configure_logging
+from core.routers.index import router as index_router
+
+logger = logging.getLogger(__name__)
+
+app = FastAPI()
+configure_logging()
```

Comment:
You can throw this call in one of the FastAPI lifecycle events upon startup

https://fastapi.tiangolo.com/advanced/events/#startup-event

Reply:
Thanks @aaronkanzer. I will look into your comments and update the code accordingly.